stable/redis/README.md

```diff
@@ -43,39 +49,89 @@ The command removes all the Kubernetes components associated with the chart and
-| `metrics.imagePullPolicy`     | Exporter image pull policy                        | `IfNotPresent`                                          |
-| `metrics.resources`           | Exporter resource requests/limit                  | Memory: `256Mi`, CPU: `100m`                            |
-| `nodeSelector`                | Node labels for pod assignment                    | {}                                                      |
-| `tolerations`                 | Toleration labels for pod assignment              | []                                                      |
-| `networkPolicy.enabled`       | Enable NetworkPolicy                              | `false`                                                 |
-| `networkPolicy.allowExternal` | Don't require client label for connections        | `true`                                                  |
-| `service.annotations`         | annotations for redis service                     | {}                                                      |
-| `service.loadBalancerIP`      | loadBalancerIP if service type is `LoadBalancer`  | ``                                                      |
-| `securityContext.enabled`     | Enable security context                           | `true`                                                  |
+| Parameter                                  | Description                                                                                                    | Default                              |               |
+|--------------------------------------------|----------------------------------------------------------------------------------------------------------------|--------------------------------------|---------------|
+| `image.registry`            | Rabbitmq Image registry                                 | `docker.io`                                              |
+| `image.repository`          | Rabbitmq Image name                                     | `bitnami/rabbitmq`                                       |
+| `image.tag`                 | Rabbitmq Image tag                                      | `{VERSION}`                                              |
+| `image.pullPolicy`          | Image pull policy                                       | `Always` if `imageTag` is `latest`, else `IfNotPresent`  |
+| `image.pullSecrets`         | Specify docker-ragistry secret names as an array        | `nil`                                                    |
+| `cluster.enabled`                          | Use master-slave topology                                                                                      | `false`                              |               |
+| `cluster.slaveCount`                       | Number of slaves                                                                                               | 1                                    |               |
```

Comment:
Should we add slave's in the default config?

Reply:
I can add one so the user sees a basic testing topology.